### PR TITLE
Usr guide mon ds docs

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

First cut at documenteation for `mon_data_space` in `eva_user_guide/data_ingest`.

### CODE REVIEW

**Comment by EdwardSafford-NOAA** [comment]
Changing merge branch.  will reissue shortly.

File: doc/eva_user_guide/data_ingest/mon_data_space.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # Monitor data space (NCEP legacy) ingest class
+Mon_data_space enables eva to ingest legacy DA monitoring data and produce plots
+similar to the legacy DA monitoring suite (https://github.com/NOAA-EMC/GSI-Monitor).
+
+The specific yaml file components necessary to accomplish this are described below.
+These example yaml files maybe be found in `eva/src/eva/tests/config`:
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/data_ingest/mon_data_space.md]: maybe be -> may be?

**Review by CoryMartin-NOAA — COMMENTED** [review]
One tiny comment

File: doc/eva_user_guide/data_ingest/mon_data_space.md (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply doc/eva_user_guide/data_ingest/mon_data_space.md]: Yup.  Got moving too fast there.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by danholdaway — APPROVED** [review]

**Pull request merged**